llvm/docs/CallGraphSection.md

```diff
@@ -0,0 +1,29 @@
+# .callgraph Section Layout
+
+The `.callgraph` section is used to store call graph information for each function, which can be used for post-link analyses and optimizations. The section contains a series of records, with each record corresponding to a single function.
```

Comment:
the post-link bit here makes me think its for Post-Link-Optimization specifically. I suppose it could be, but that isn't the primary motivation we have ATM. maybe tweak the verbiage a bit?

Reply:
Removed the part about post-link analyses.